### PR TITLE
[CHI-340] Add -bg--muted color utility

### PR DESCRIPTION
**Pull request by davidojedacl**
@mattnickles

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/229/ for more details.

File: src/custom-elements/docs/docs.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "timestamp": "2019-07-22T15:28:05",
+  "timestamp": "2019-08-20T10:36:44",
```
Comment (mattnickles) [inline src/custom-elements/docs/docs.json]: @davidojedacl can you remove this file from your PR?

Comment (davidojedacl) [reply src/custom-elements/docs/docs.json]: i think is necessary after make tests

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/241/ for more details.

**Pull request merged**